pkg/cloudevents/event_reader.go

```diff
@@ -81,15 +81,15 @@ func (e Event) DataMediaType() string {
 	return ""
 }
 
-// DataContentEncoding implements EventReader.DataContentEncoding
-func (e Event) DataContentEncoding() string {
+// DeprecatedDataContentEncoding implements EventReader.DeprecatedDataContentEncoding
+func (e Event) DeprecatedDataContentEncoding() string {
 	if e.Context != nil {
-		return e.Context.GetDataContentEncoding()
+		return e.Context.DeprecatedGetDataContentEncoding()
 	}
 	return ""
 }
 
-// DataContentEncoding implements EventReader.DataContentEncoding
```

Comment:
Likely this was an earlier type since it should be for Extensions().

Reply:
Fixed, nice catch.